test/Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests/project.json

```diff
@@ -1,22 +1,18 @@
 {
   "dependencies": {
     "xunit": "2.1.0",
-    "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.0.0-*"
+    "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.0.0-*",
+    "NETStandard.Library": "1.5.0-*"
   },
   "frameworks": {
-    "netcoreapp1.0": {
```

Comment:
I sent a PR to KoreBuild to recognize netcoreapp1.0 test projects. I was on yesterday's build and that's why I was seeing this: Deps file [C:\Source\IISIntegration\test\dotnet-publish-iis.Tests\bin\Debug\netcoreapp1.0\win7-x64\dotnet-publish-iis.Tests.deps.json] specified but doesn't exist  (and also a reason of sending the email where I propose automatic update of dotnet cli if we took a newer version)